File: submissions/GIA999/problem2.java
import java.io.*;
class problem2
{
public static void main(String args [])throws IOException
{
InputStreamReader read = new InputStreamReader(System.in);
BufferedReader ob = new BufferedReader(read);
int t = Integer.parseInt(ob.readLine ());
for (int i=0;i<t;i++)
{
int a = Integer.parseInt(ob.readLine ());
System.out.println( LCM(a)+a);
}
}

static long LCM(int m)
{
long s = 1;
for (int j=2;j<=Math.sqrt(m);j++)
{
if ((m%j)==0)
s = s + j+(m/j);
}
if(Math.sqrt(m)==Math.floor(Math.sqrt(m)))s-=Math.sqrt(m);
return s;
}
}

File: submissions/GIA999/problem2.cpp
#include<iostream>
#include<math.h>
using namespace std;
int SumDivs(int n)
{
   int sum = 1;
   double m = sqrt(n);
   for (int i = 2; i <= m; i++)
      if(n % i == 0) sum += (i + n / i);
   return sum;
}
int main()
{
 int a;
 cin>>a;
 long int s[a];
 for(int k=0;k<a;k++)
 {
    cin>>s[k];
  cout<<SumDivs(s[k])+s[k]<<endl;
 }


 return 0;
}
